POLIO-760 Upgrade Enketo version used for local development

olethanh opened the pull request:
Instruct docker-compose to use the new image 

This is part of the broader ticket on upgrading Enketo andwas mainly done to check that ware compatible with the new Enketo version and can deploy it on servers. No change in behaviour is expected


## How to test

1. Do a `docker-compose pull` to retrieve the new enketo image.
2. Launch your docker compose with Enketo, see the Enketo sections in the main README.md for instruction on how to do this.
3. Check that you can still create and edit submission. Try with different Form type

quang-le reviewed (approved):
Front-end looks fine to me

olethanh commented on hat/assets/js/apps/Iaso/domains/orgUnits/utils.js:
```diff
@@ -66,9 +66,14 @@ export const getSourcesWithoutCurrentSource = (
-        if (orgUnit.org_unit_type_name && withType) {
-            message += ` (${orgUnit.org_unit_type_name})`;
+        if (orgUnit.name) {
+            message = `${orgUnit.name}${
+                orgUnit.source ? ` - source: ${orgUnit.source}` : ''
+            }${
+                orgUnit.org_unit_type_name && withType
+                    ? ` (${orgUnit.org_unit_type_name})`
+                    : ''
```
it's like an  escape in an escape in a escape. I've got a feeling there is probably a more readable way to write this?

olethanh replied:
maybe something like
```
orgUnit.name \
   + orgunit.source ? ` - source: ${orgUnit.source}` : '' \
   + orgUnit.org_unit_type_name && withType?  ` (${orgUnit.org_unit_type_name})` : ''
```

olethanh reviewed (changes requested):
We should probably forbid empty name in the futur but that's probably a good fix in the mean time

quang-le commented on hat/assets/js/apps/Iaso/domains/orgUnits/utils.js:
```diff
@@ -63,18 +63,19 @@ export const getSourcesWithoutCurrentSource = (
 
-export const getOrgunitMessage = (orgUnit, withType) => {
+export const OrgunitMessage = ({ orgUnit, withType }) => {
     let message = textPlaceholder;
```
name should be orgunitMessage, since it's not a class

olethanh committed:
POLIO-760 Upgrade Enketo version used for local development

This use the new image. This was mainly done to check that ware compatible with the new Enketo version and can deploy it on server
